### PR TITLE
chore(deps): update mongosh to latest

### PR DESCRIPTION
**Pull request by gribnoysup**
Bump all mongosh packages to latest

### CODE REVIEW

File: package-lock.json
Comment (gribnoysup) [inline package-lock.json:1]: Looks like a big change, but all those look legit, caused by aws-sdk bump

**Comment by paula-stacho** [comment]
I've just tried this on my branch and looks like there are some breaking changes
new shell export -> https://github.com/mongodb-js/mongosh/commit/61d7214fa62e49af36039826e90dba8b7b84dc18#diff-2e66ef480f7514a1984804cee2d4c93cc426c70c1b310fed99579ba55e4b08dfR465-R470

https://github.com/mongodb-js/compass/blob/3490eefae1bbd7f506337410b5153428a50962a6/packages/compass-shell/src/components/compass-shell/compass-shell.tsx#L5

File: packages/compass-shell/src/plugin.tsx [hunk 1]
```diff
@@ -11,6 +11,9 @@ import type { DataService } from '@mongodb-js/compass-connections/provider';
 import type { PreferencesAccess } from 'compass-preferences-model';
 import { usePreference } from 'compass-preferences-model/provider';
 import type { TrackFunction } from '@mongodb-js/compass-telemetry';
+import { Theme, ThemeProvider } from '@mongodb-js/compass-components';
+
+const SHELL_THEME = { theme: Theme.Dark, enabled: true };
```
Comment (paula-stacho) [inline packages/compass-shell/src/plugin.tsx:16]: what is the enabled setting?

**Review by paula-stacho — APPROVED** [review]
🎉

**Comment by gribnoysup** [comment]
Redid last commit because we spotted legit issue in `win-export-certificate-and-key` for electron and @addaleax did a quick fix for that

**Pull request merged**